src/btree.c

```diff
@@ -3083,13 +3083,66 @@ btree_print_offset_table(platform_log_handle *log_handle, btree_hdr *hdr)
    platform_log(log_handle, "\n");
 }
 
+static void
+btree_print_btree_pivot_stats(platform_log_handle *log_handle,
+                              btree_pivot_stats   *pivot_stats)
+{
+   // If nothing has been updated, yet, print a msg accordingly.
+   if ((pivot_stats->num_kvs == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->key_bytes == BTREE_UNKNOWN_COUNTER)
+       && (pivot_stats->message_bytes == BTREE_UNKNOWN_COUNTER))
+   {
+      platform_log(log_handle, "   (Pivot Stats=BTREE_UNKNOWN_COUNTER)\n");
+      return;
```

Comment:
This little chunk will come ahead of this PR under another in-flight change-set for PR #521 . (Ignore this diff for this review.)